android/src/main/java/com/magicleap/magicscript/scene/nodes/dropdown/UiDropdownListNode.kt

```diff
@@ -151,25 +160,37 @@ class UiDropdownListNode(
         }
     }
 
+    private fun setMultiSelect(props: Bundle) {
+        val multiSelect = props.read<Boolean>(PROP_MULTI_SELECT)
+        if (multiSelect != null) {
+            listNode.multiSelect = multiSelect
+        }
+    }
+
     private fun setShowList(props: Bundle) {
         if (props.containsKey(PROP_SHOW_LIST)) {
             val show = props.getBoolean(PROP_SHOW_LIST)
             if (show && !listNode.isVisible) {
                 showList()
-            } else if (listNode.isVisible) {
+            } else if (!show && listNode.isVisible) {
                 hideList()
             }
         }
     }
 
+    private fun setMaxListHeight(props: Bundle) {
+        val maxListHeight = props.read<Double>(PROP_LIST_MAX_HEIGHT)
+        if (maxListHeight != null) {
```

Comment:
come to the dark side and use 
```
props.read<Double>(PROP_LIST_MAX_HEIGHT)?.let {
...
}
```
![](https://media.giphy.com/media/GIIC4jmmUlXZS/giphy.gif)